DataSetExplorer/DataSetBuilder/Model/CandidateDataSetInstance.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+namespace DataSetExplorer.DataSetBuilder.Model
+{
+    public class CandidateDataSetInstance
```

Comment:
We need a better name. First, I would like to remove the "DataSet" keywords from most of the model. We need it for the high-level DataSet, and I think it makes sense to have the "DataSetProject". We should remove it from this and the pure Instance class.

Now, regarding this class name, a better alternative might be `SmellCandidateInstances` or `CandidateSmellInstances` or `SmellInstanceGroup`. Maybe the first one, as Candidate is a notable name in the domain of annotation.

As a side note, this will be an interesting point for your paper, where the expanded conceptual model is described and the reasoning behind it. Maybe create a google doc or Clickup card where you will keep track of all the things you'll discuss in your paper.